Makefile

```diff
@@ -407,3 +407,44 @@ consistent-crds-manifests-upstream:
 	$(Q)cd ./manifests-upstream/${OPERATOR_VERSION}/ && ln -srf ../../deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml \
 	servicebindingrequests.apps.openshift.io.crd.yaml
 
+## -- Target for merge to master dev release --
+.PHONY: merge-to-master-release
+## Make a dev release on every merge to master
+merge-to-master-release:
+	echo "${QUAY_TOKEN}" | docker login -u "redhat-developer+travis" --password-stdin quay.io
+	$(eval COMMIT_COUNT := $(shell git rev-list --count HEAD))
+	$(Q)operator-sdk build \
+	"$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)"
+	#docker login -u="redhat-developer+travis" -p=${QUAY_TOKEN}
+	docker tag "$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)" \
+	"$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)"
+	docker push "$(OPERATOR_IMAGE_REL):$(GIT_COMMIT_ID)"
+
+
+## -- Target for pushing manifest bundle to service-binding-operator-manifest repo --
+.PHONY: push-to-manifest-repo
+## Push manifest bundle to service-binding-operator-manifest repo
+push-to-manifest-repo:
```

Comment:
Where are we copying over the rolebinding/role information from `deploy/` to the csv's `permissions` section?

Reply:
the file under deploy/olm-catalog/csv-config.yaml does that 
```
crd-cr-paths:
- deploy/crds
operator-path: deploy/operator.yaml
role-paths:
- deploy/role.yaml
```
whenever a new generate happens it copies roles from deploy/roles.yaml
reference https://github.com/operator-framework/operator-sdk/blob/master/doc/design/milestone-0.2.0/csv-generation.md#configuration

Reply:
Ah, nice.